llvm/test/tools/llvm-symbolizer/skip-line-zero.s

```diff
@@ -0,0 +1,247 @@
+##  Test  --skip-line-zero option.
+##
+##  This test uses handcrafted assembly to produce the following line table:
+##  Address            Line   Column File   ISA Discriminator OpIndex Flags
+##  ------------------ ------ ------ ------ --- ------------- ------- -------------
+##  0x0000000000001710      1      0      1   0             0       0
+##  0x0000000000001714      0      0      1   0             0       0
+##  0x0000000000001719      1      0      1   0             0       0
+##  0x000000000000171b      1      0      1   0             0       0  end_sequence
```

Comment:
I would make the line number for each of the non-zero lines unique. That way we can be sure that --skip-line-zero has selected the right row from the table from the line number returned.

Reply:
Good point about the `end_sequence`s, thanks. I made this comment because with the current table the output for the first sequence will always be line = 1, column = 0 no matter which non-zero row is selected. Actually, we don't need to set unique line numbers as I originally suggested. If we just set the column for the 0x1719 row to a be non-zero then a result of line = 1, column = 0 must have come from the first row for the first sequence.

Reply:
I have made a code suggestion for this now.